grantnav/frontend/templates/search_widgets.html

```diff
@@ -211,18 +224,20 @@ <h1>Search input is not valid</h1>
   const selectViewElement = document.getElementById('select-views');
   const selectWidthElement = document.getElementById('select-width');
   const inputWidthElement = document.getElementById('input-width');
+  const inputRowsElement = document.getElementById('input-rows');
 
-  if (selectViewElement && selectWidthElement) {
+  if (selectViewElement && selectWidthElement && inputWidthElement && inputRowsElement) {
     const selectedView = selectViewElement.value;
     const widthType = selectWidthElement.value;
     const width = inputWidthElement.value;
+    const rows = inputRowsElement.value;
 
     const preview = document.getElementById('sample-frame');
     const code = document.getElementById('generated-code');
 
-    code.innerHTML = `&lt;iframe src="${window.location.origin}/widget-render/${selectedView}${window.location.search}" style="height: 700px; width: ${width}${widthType};" frameBorder="0"&gt;&lt;/iframe&gt;`;
+    code.innerHTML = `&lt;iframe src="${window.location.origin}/widget-render/${selectedView}${window.location.search}?rows=${rows}" style="height: 700px; width: ${width}${widthType};" frameBorder="0"&gt;&lt;/iframe&gt;`;
 
-    preview.src = `/widget-render/${selectedView}${window.location.search}`;
+    preview.src = `/widget-render/${selectedView}${window.location.search}?rows=${rows}`;
```

Comment:
Not sure this looks right, a resulting search string would be something like:

`'?query=wef&default_field=%2A&sort=_score+desc?rows=3'` 

Whereas what I think would be desired is

`'?query=wef&default_field=%2A&sort=_score+desc&rows=3'`

Reply:
Will `window.location.search` always have some query params? If so, I can just change the `?` to a `&`. If not I'll put some conditional in there.

Reply:
Unfortunately that can't be relied on. You could do an `if` conditional but might be better to use [URLSearchParams]( https://developer.mozilla.org/en-US/docs/Web/API/URLSearchParams )